__tests__/app.test.js

```diff
@@ -442,3 +442,116 @@ describe("GET /api/users", () => {
             });
     });
 });
+
+describe("GET /api/articles (QUERIES)", () => {
+    describe("GET /api/articles?topic=mitch", () => {
+        test("200: should respond with an array of all articles belonging to a specific topic sorted by date (default), the default order is descending", () => {
+            return request(app)
+                .get("/api/articles?topic=mitch")
+                .expect(200)
+                .then(({ body }) => {
+                    const { articles } = body;
+
+                    expect(articles).toBeInstanceOf(Array);
+                    expect(articles).toHaveLength(12);
+                    expect(articles).toBeSortedBy("created_at", { descending: true });
+
+                    articles.forEach((article) => {
+                        expect(article).toMatchObject({
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            article_id: expect.any(Number),
+                            topic: "mitch",
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            article_img_url: expect.any(String),
+                            comment_count: expect.any(String),
+                        });
+                    });
+                });
+        });
+
+        test("400: should respond with 'Bad request' if the provided topic value does not exist", () => {
+            return request(app)
+                .get("/api/articles?topic=nonsense")
+                .expect(400)
```

Comment:
This should really be a 404 -> technically 'nonsense' could be a topic! It just isn't one!